cypress/integration/AlertBar.spec.js

```diff
@@ -0,0 +1,59 @@
+/* global cy */
+
+import { AlertBar } from '../../src'
+
+describe('AlertBar - Functional', () => {
```

Comment:
We are not always going to be adding the word "Functional" here right?

Reply:
I just didn't know a better word for describing the difference between functional and visual tests

Reply:
Aha OK makes sense